web/templates/index.html

```diff
@@ -255,50 +285,54 @@ <h5 class="card-title">Filters</h5>
-                  </button>
-                  <button id="library-rescan-btn" type="button" class="btn btn-secondary mr-1">
-                    <i class="fas fa-sync-alt" aria-hidden="true"></i> Rescan Files
-                  </button>
-                  <button id="library-download-btn" type="button" class="btn btn-secondary mr-1">
-                    <i class="fas fa-download" aria-hidden="true"></i> Download All
-                  </button>
-                  <button type="button" class="btn btn-danger mr-1" data-toggle="modal" data-target="#deleteWarningModal">
-                    <i class="fas fa-trash-alt" aria-hidden="true"></i> Delete All
-                  </button>
+                    <button id="add-to-playlist-btn" type="button" class="btn btn-secondary mr-1">
+                        <i class="fas fa-plus" aria-hidden="true"></i> Add All
+                    </button>
+                    <button id="library-rescan-btn" type="button" class="btn btn-secondary mr-1">
+                        <i class="fas fa-sync-alt" aria-hidden="true"></i> Rescan Files
+                    </button>
+                    <button id="library-download-btn" type="button" class="btn btn-secondary mr-1">
+                        <i class="fas fa-download" aria-hidden="true"></i> Download All
+                    </button>
+                    <button type="button" class="btn btn-danger mr-1" data-toggle="modal"
+                        data-target="#deleteWarningModal">
+                        <i class="fas fa-trash-alt" aria-hidden="true"></i> Delete All
+                    </button>
                 </div>
 
-                <div class="modal fade" id="deleteWarningModal" tabindex="-1" role="dialog" aria-labelledby="Warning-Delete-File" aria-hidden="true">
-                  <div class="modal-dialog" role="document">
-                    <div class="modal-content">
-                      <div class="modal-header">
-                        <h5 class="modal-title" id="deleteWarningModalLabel">Are you really sure?</h5>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
-                          <span aria-hidden="true">&times;</span>
-                        </button>
-                      </div>
-                      <div class="modal-body">
-                        All files listed here, include files on other pages, will be deleted from your
-                        hard-drive.
-                        Is that what you want?
-                      </div>
-                      <div class="modal-footer">
-                        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-                        <button id="library-delete-btn" type="button" class="btn btn-danger" data-dismiss="modal">Delete All Listed Files</button>
-                      </div>
+                <!-- QUESTION: should this div have aria-hidden as true?? -->
```

Comment:
I think the answer is "no" since it has `display: none` set unless being triggered, which means it won't be included in the accessibility tree by default.

Reply:
Oh, that's true. Excellent :)